### DISCUSSION TITLE
Go to definition not work

### DISCUSSION THREAD

**Discussion by JuoCode** [LazyVim/LazyVim#3163] [Q&A] [answered]
Hello everyone, I'm using LazyVim to write TypeScript and React code, and today my editor has been acting up. Some symbols aren't jumping to their definitions, and after adding print(item) in telescope/builtin/__lsp.lua:187, I noticed that item.user_data is nil. I'm not sure what's going on. Can anyone please help me out here? I'd really appreciate it.

**Comment by folke** [comment, marked as answer]
Update telescope

**Reply by JuoCode** [reply]
Thank you, its work after update telescope.
